cpp/modmesh/math/pymod/wrap_Complex.cpp

```diff
@@ -60,7 +60,7 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapComplex
             .def(
                 py::init(
                     []()
-                    { return std::make_shared<wrapped_type>(); }))
+                    { return std::make_shared<wrapped_type>(wrapped_type{0.0, 0.0}); }))
```

Comment:
To avoid different behaviors across different compilers, I assigned initial values to the Python constructor without any arguments

Reply:
This is a benign discrepancy between Python and C++ code.